sdk/daml-lf/archive/src/main/protobuf/com/digitalasset/daml/lf/archive/daml_lf2.proto

```diff
@@ -269,6 +280,7 @@ message Type {
     sint64 nat = 6;
     Syn syn = 7;
     int32 interned_type = 8;
+    TApp tapp = 9;
```

Comment:
We should add a comment explaining what feature this constructor was added for, and mark it as 2.dev for now.